www/js/pages/profile/profile.controller.js

```diff
@@ -68,14 +68,13 @@
             });
 
         vm.init = () => {
-            vm.isVirtualAccount = !!appStateService.virtuality;
             const account = accountService.getDefault();
-            const landingCompany = localStorage.getItem('landingCompany');
+            const landingCompany = account.landing_company_name || localStorage.getItem('landingCompany');
+            vm.isVirtualAccount = account.landing_company_name === 'virtual' || !!appStateService.virtuality;
```

Comment:
@nazaninreihani 
It's better to use `clientService.isAccountOfType` instead of manually check it. in this way, if you want to change the logic, easily you just change that function logic.

Reply:
@morteza-binary clientService.isAccountOfType checks between account types (virtual, real, financial/gaming), here we need to check if landing company is the same as what we want or not, I don't think these are the same, but we can have a function in clientjs and use them for all

Reply:
@nazaninreihani 
Yes, You're right, as you mentioned it's more convenient to define a function for it in `client.service` and use it.